core/src/main/java/org/jclouds/concurrent/config/ExecutorServiceModule.java

```diff
@@ -113,7 +153,7 @@ protected void configure() { // NO_UCD
    @Provides
    @Singleton
    final TimeLimiter timeLimiter(@Named(PROPERTY_USER_THREADS) ListeningExecutorService userExecutor) {
-      return new SimpleTimeLimiter(userExecutor);
+      return createSimpleTimeLimiter(userExecutor);
```

Comment:
Could you also make changes to `BaseRestApiExpectTest` and `BaseRestApiTest.java` for consistency?

Reply:
Done